src/python/turicreate/toolkits/_data_zoo.py

```diff
@@ -21,14 +22,21 @@ class OneShotObjectDetectorBackgroundData(object):
     def __init__(self):
         self.source_tar_filename = "one_shot_backgrounds.sarray.tar"
         self.destination_tar_filename = "one_shot_backgrounds.sarray.tar"
+        self.destination_sarray_filename = "one_shot_backgrounds.sarray"
+        self.destination_sarray_path = _os.path.join(_get_cache_dir("data"),
+            self.destination_sarray_filename)
         self.sarray_url = _urlparse.urljoin(
             DATA_URL_ROOT, self.source_tar_filename)
         self.sarray_url_md5_pairs = [
             (self.sarray_url, "08830e90771897c1cd187a07cdcb52b4")
             ]
 
     def get_backgrounds_path(self):
-        backgrounds_path = _download_and_checksum_files(
+        if _os.path.exists(self.destination_sarray_path):
+            return self.destination_sarray_path
```

Comment:
How do we know the contents of this path are valid?

Reply:
Added a check!